src/libs/ReportUtils.js

```diff
@@ -1041,6 +1041,16 @@ function getReportName(report) {
     return _.map(participantsWithoutCurrentUser, (login) => getDisplayNameForParticipant(login, isMultipleParticipantReport)).join(', ');
 }
 
+/**
+ * Get the report for a reportID
+ *
+ * @param {String} reportID
+ * @returns {Object}
+ */
+function getReport(reportID) {
+    return allReports[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`];
+}
```

Comment:
This also causing crash if user is does is not authenticated yet, and does not have any value in onyx. Part of https://github.com/Expensify/App/issues/20394
<img width="1440" alt="Screenshot 2023-06-08 at 12 08 35 AM" src="https://github.com/Expensify/App/assets/59907218/f7eb4030-f68d-47e6-805a-daef3b3e9e77">

Reply:
FYI this can still return null if the report key is present, but the report was deleted (set to null)